analyzers/src/Directory.Build.targets

```diff
@@ -7,6 +7,18 @@
     <BinariesFolder>$(MSBuildThisFileDirectory)\..\packaging\binaries\</BinariesFolder>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Using Include="System" />
```

Comment:
WDYT about moving the shared usings from tests and src to analyzer/Directory.Build.targets?